src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public void doWork() {

```

Comment:
```suggestion
    @Override
    public void doWork() {
```

for all inherited methods use this annotation